ArduCopter/Log.cpp

```diff
@@ -547,8 +585,24 @@ const struct LogStructure Copter::log_structure[] = {
 // @Field: aY: Target acceleration, Y-Axis
 // @Field: aZ: Target acceleration, Z-Axis
 
-    { LOG_GUIDEDTARGET_MSG, sizeof(log_GuidedTarget),
-      "GUID",  "QBfffbffffff",    "TimeUS,Type,pX,pY,pZ,Terrain,vX,vY,vZ,aX,aY,aZ", "s-mmm-nnnooo", "F-BBB-BBBBBB" , true },
+    { LOG_GUIDED_POSITION_TARGET_MSG, sizeof(log_Guided_Position_Target),
+      "GPOS",  "QBfffbffffff",    "TimeUS,Type,pX,pY,pZ,Terrain,vX,vY,vZ,aX,aY,aZ", "s-mmm-nnnooo", "F-BBB-BBBBBB" , true },
+
+// @LoggerMessage: GATT
+// @Description: Guided mode attitude target information
+// @Field: TimeUS: Time since system startup
+// @Field: Type: Type of guided mode
+// @Field: Roll: Target attitude, Roll
+// @Field: Pitch: Target attitude, Pitch
+// @Field: Yaw: Target attitude, Yaw
+// @Field: RollRt: Roll rate
+// @Field: PitchRt: Pitch rate
+// @Field: YawRt: Yaw rate
+// @Field: Thrust: Thrust 
+// @Field: ClimbRt: Climb rate
+
+    { LOG_GUIDED_ATTITUDE_TARGET_MSG, sizeof(log_Guided_Attitude_Target),
+      "GATT",  "QBffffffff",    "TimeUS,Type,Roll,Pitch,Yaw,RollRt,PitchRt,YawRt,Thrust,ClimbRt", "s-dddkkk-n", "F--------B" , true },
```

Comment:
Nitpick:
The difference between `-` and `0` is subtle. The first means a multiplier does not apply and the other means `x1`. The result is the same code wise, but interpretation is different. 

```suggestion
    { LOG_GUIDED_ATTITUDE_TARGET_MSG, sizeof(log_Guided_Attitude_Target),
      "GATT",  "QBffffffff",    "TimeUS,Type,Roll,Pitch,Yaw,RollRt,PitchRt,YawRt,Thrust,ClimbRt", "s-dddkkk-n", "F-000000-B" , true },
```